p2p/synch/getblockdatas.go

```diff
@@ -89,14 +90,16 @@ func (s *Sync) getBlockDataHandler(ctx context.Context, msg interface{}, stream
 		if err != nil {
 			return err
 		}
-		bds = append(bds, &pb.BlockData{BlockBytes: blocks})
+		pbbd := pb.BlockData{BlockBytes: blocks}
+		if uint64(bd.SizeSSZ()+pbbd.SizeSSZ()+4) >= s.p2p.Encoding().GetMaxChunkSize() {
```

Comment:
why the magic number 4 ? please name the const instead of using magic number.

Reply:
https://github.com/Qitmeer/qitmeer/pull/406/commits/369220c75f26efeca3385ca0f277d9e7d6c8c61b